pbs_light_test.go

```diff
@@ -28,11 +28,10 @@ func forceEnv(t *testing.T, key string, val string) func() {
 				t.Fatalf("Error unsetting evnvironment %s", key)
```

Comment:
I realize you didn't misspell this word but, could you correct it? Unless it is a hassle to go through another round of reviews. If so, we can live with it.

Reply:
No worries I’ll handle it !

Reply:
Here you go: #996